frame/vesting/src/lib.rs

```diff
@@ -648,6 +653,34 @@ impl<T: Config> Pallet<T> {
 
 		Ok((schedules, locked_now))
 	}
+
+	/// Ensure the correctness of the state of this pallet.
+	///
+	/// This should be valid before or after each state transition of this pallet.
+	///
+	/// ## Invariants:
+	///
+	/// * The `per_block` payments left of every account currently vesting
+	/// * is equal to the total balance currently locked.
+	#[cfg(any(feature = "try-runtime", test))]
+	pub fn do_try_state() -> Result<(), &'static str> {
+		Vesting::<T>::iter().for_each(|(_, d)| {
+			let vesting = d.to_vec();
+			let mut total_per_block: BalanceOf<T> = Zero::zero();
+			let mut total_locked_now: BalanceOf<T> = Zero::zero();
```

Comment:
I suggest using more descriptive names for these, something like:

```suggestion
			let mut total_to_vest: BalanceOf<T> = Zero::zero();
			let mut total_locked: BalanceOf<T> = Zero::zero();
```

Reply:
I think `total_per_block` is descriptive (i.e total amount left to be unlocked after the starting block), want users to know it's referring to `VestingInfo.per_block`.

Reply:
`total_per_block` makes me think of "how much is going to vest every block" instead of "what is the total amount left to vest over every remaining block"

Reply:
I see what you mean